examples/pets/tests/createAndShare.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { test, expect } from "@playwright/test";
+
+test("create a new post and share", async ({ page }) => {
+    await page.goto("/");
+
+    await page.getByRole("textbox").fill("S. Mario");
+
+    await page
+        .getByRole("button", {
+            name: "Sign Up as new account",
+        })
+        .click();
+
+    await page
+        .getByRole("link", {
+            name: "New Post",
+        })
+        .click();
+
+    await page.getByPlaceholder("Pet name").fill("Yoshi");
+
+    // Start waiting for file chooser before clicking. Note no await.
+    const fileChooserPromise = page.waitForEvent("filechooser");
+
+    await page.getByTestId("file-upload").click();
+
+    const fileChooser = await fileChooserPromise;
+    await fileChooser.setFiles("./public/jazz-logo.png");
+
+    await page
+        .getByRole("button", {
+            name: "Submit Post",
+        })
+        .click();
+
+    expect(
+        page.getByRole("heading", {
+            name: "Yoshi",
+        }),
+    ).toBeVisible();
+
+    await page
+        .getByRole("button", {
+            name: "Share",
+        })
+        .click();
+
+    await page
+        .getByRole("button", {
+            name: "Log out",
+        })
+        .click();
+
+    await page.getByRole("textbox").clear();
+    await page.getByRole("textbox").fill("Luigi");
+
+    await page
+        .getByRole("button", {
+            name: "Sign Up as new account",
+        })
+        .click();
+
+    const invitation = await page.evaluate(() =>
+        navigator.clipboard.readText(),
+    );
+
+    await page.goto(invitation);
+    await page.reload();
```

Comment:
Had to reload here in order to make the invitation work 🤔 

Can this be considered a bug?

Reply:
Also the CI action always fail on the first run here and succeds on the first retry:

https://github.com/gardencmp/jazz/actions/runs/10646621520/job/29513668210#step:7:1

Probably related to [JAZZ-74: Ensure invites are synced before considering them ready](https://linear.app/garden-computing/issue/JAZZ-74/ensure-invites-are-synced-before-considering-them-ready)